pkg/apis/eksctl.io/v1alpha5/assets/schema.json

```diff
@@ -1319,11 +1325,6 @@
         "desiredCapacity": {
           "type": "integer"
         },
-        "disableASGTagPropagation": {
```

Comment:
Why is this deleted? 🤔

Reply:
I thought, since it was deprecated, that the doc could be removed to avoid people using it. I realise now it was not done for the non-managed nodegroup. I did put it back then to keep consistent.

Reply:
Yeah, we leave things around for a little while before deleting. :) It's a deprecation tactic.